DDCore/src/plugins/DetectorChecksum.cpp

```diff
@@ -515,6 +515,43 @@ const DetectorChecksum::entry_t& DetectorChecksum::handleSolid(Solid solid) cons
     else if ( shape->IsA() == TGeoShapeAssembly::Class() )   {
       log << "<shape_assembly " << nam << "\"/>";
     }
+    else if ( shape->IsA() == TGeoTessellated::Class() )  {
+      const TGeoTessellated* sh  = (TGeoTessellated*)shape;
+      log << "<define>" << newline;
+      if ( sh->IsClosedBody() == false ) {
+        except("DetectorChecksum","+++ TGeoTessellated volume is not closed: %s", solid.name());
+      }
+      for (int ivertex = 0; ivertex < sh->GetNvertices(); ivertex++)  {
+        // Note: const_cast since TGeoTessellated::GetVertex not marked const in ROOT <= 6.28
+        const auto& vtx = const_cast<TGeoTessellated*>(sh)->GetVertex(ivertex);
+        log << "<position name\"" << nam << "_v" << ivertex
+            << " lunit=\"" << m_len_unit_nam << "\""
+            << " x=\"" << vtx.x() << "\""
+            << " y=\"" << vtx.y() << "\""
+            << " z=\"" << vtx.z() << "\""
```

Comment:
Ensure correct units.
```suggestion
            << " x=\"" << vtx.x()/m_len_unit << "\""
            << " y=\"" << vtx.y()/m_len_unit << "\""
            << " z=\"" << vtx.z()/m_len_unit << "\""
```